examples/sht31_test.py

```diff
@@ -0,0 +1,29 @@
+import time
+import board
+import busio
+import adafruit_sht31
+
+# Create library object using our Bus I2C port
+i2c = busio.I2C(board.SCL, board.SDA)
+sensor = adafruit_sht31.SHT31(i2c)
+
+
+while True:
+    print(sensor.temperature_and_relative_humidity)
+    print(sensor.temperature)
```

Comment:
For the example code, please mimic
https://github.com/adafruit/Adafruit_CircuitPython_SI7021/blob/master/examples/main.py
where we print text with each reading, just temp and humidity.

if you want to, maybe every 10 readings you can 'turn on the heater' for 1 second?

Reply:
I removed sht31_test and did this in sht31_simpletest.py.